controllers/workspace/status.go

```diff
@@ -92,9 +92,11 @@ func syncConditions(workspaceStatus *dw.DevWorkspaceStatus, currentStatus *curre
 		currCondition, ok := currentStatus.conditions[workspaceCondition.Type]
 		if !ok {
 			// Didn't observe this condition this time; set status to unknown
```

Comment:
is this up to date?

Reply:
Yeah, the idea is that if a workspace had the `Ready` condition set to `true`, but this reconcile ended before we confirmed the workspace was still running, we want to set this condition to `Unknown` for now (since it was not observed). 

The difference in this PR is to not update that `Unknown` conditions `LastTransitionTime` if it's not actually transitioning.

Another option is deleting `Unknown` conditions from the list, which would do a similar thing, but does lose _some_ information.